Packs/McAfee_ESM/ReleaseNotes/1_1_5.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### McAfee ESM v2
+- Fixed an issue where ***raw_response*** argument returned final result instead of the raw data.
```

Comment:
```suggestion
- Fixed an issue where ***raw_response*** argument returned final result instead of the raw data.
- Updated the Docker image to: *demisto/python3:3.9.2.17957*.
```